jme3-core/src/main/java/com/jme3/anim/MorphTrack.java

```diff
@@ -113,24 +132,33 @@ public void setTimes(float[] times) {
 
 
     /**
-     * Set the weight for this morph track
+     * Sets the weights for this morph track
      *
-     * @param times    a float array with the time of each frame (alias created
-     *                 -- do not modify after passing it to this setter)
-     * @param weights  the weights of the morphs for each frame (alias created
-     *                 -- do not modify after passing it to this setter)
+     * @param times        a float array with the time of each frame
```

Comment:
The "alias create" comment applies here, too.

Reply:
Noted and applied both here and to the new `weights` parameter of `setNbMorphTargets()`.